playground/TicTacToe.py

```diff
@@ -0,0 +1,81 @@
+import os
+os.system('clear')
+
+class Board:
+    def __init__(self):
+        self.cells=[
+        [" "," "," "]
+        ,[" "," "," "]
+        ,[" "," "," "]
+    ]
+        n=len(self.cells)
+        self.row_container=[0]*n
+        self.col_container=[0]*n
+        self.diag_container=[0]*n
+        self.antiDiag_container=[0]*n
+
+    def display(self):
+        for i,cell in enumerate(self.cells):
+            print(*cell, sep='|')
+            if i<len(self.cells)-1:
+                print('-----')
+
+    def update_cell(self,cell_num,player):
+        if cell_num < 10:
+            r = (cell_num-1)//3
+            c = (cell_num-1)%3
+        else:
+            raise Exception("Number not valid")
+        if self.cells[r][c]==" ":
+            self.cells[r][c] = player
+            if self.is_winner(r,c):
```

Comment:
Two things done by this function update_cell - ideally, only one action can be taken by the update_cell() and have the other method calls in the main